flow/core/experiment.py

```diff
@@ -153,6 +166,24 @@ def rl_actions(*_):
                 vel.append(np.mean(self.env.k.vehicle.get_speed(veh_ids)))
                 ret += reward
 
+                # collect additional information for the data pipeline
+                for vid in veh_ids:
```

Comment:
Would be good to move this to a separate method that can be called from multiple different scripts.

Reply:
agreed